src/Microsoft.Azure.NotificationHubs.DotNetCore.Tests/NotificationContentTypeTest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.Azure.NotificationHubs.DotNetCore.Tests
+{
+    public class NotificationContentTypeTest
+    {
+        [Fact]
+        public void NotificationContentTypeCorrespondsToApi()
+        {
+            Assert.Equal("application/json", new AppleNotification("{\"aps\":{\"alert\":\"alert!\"}}").ContentType);
+            Assert.Equal("application/json", new FcmNotification("{\"data\":{\"message\":\"Message\"}}").ContentType);
+            Assert.Equal("application/xml", new MpnsNotification("<wp:Notification xmlns:wp=\"WPNotification\" Version=\"2.0\"><wp:Tile Id=\"TileId\" Template=\"IconicTile\"><wp:Title Action=\"Clear\">Title</wp:Title></wp:Tile></wp:Notification>").ContentType);
```

Comment:
As noted in ]our own documentation](https://docs.microsoft.com/en-us/rest/api/notificationhubs/send-mpns-native-notification) it should be application/xml;charset=utf-8